### PR TITLE
Feature/input

### PR DESCRIPTION
**Pull request by ArzuuZholdosheva**

### CODE REVIEW

File: src/assets/icons/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { ReactComponent as ErrorIcon } from './ErrorIcon.svg'
+
```
Comment (NuraiymMamatova) [inline src/assets/icons/index.js]: we do only export without import

File: src/components/UI/input/Input.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            />
+         </>
+      )
+   }
+)
+
+const StyledFormHelperText = styled(FormHelperText)(({ valid }) => ({
+   fontFamily: 'Inter',
+   fontWeight: 400,
+   fontSize: '14px',
```
Comment (NuraiymMamatova) [inline src/components/UI/input/Input.jsx]: use relative units

File: src/components/UI/input/Input.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+         onBlur,
+         valid,
+         borderError,
+         ...rest
+      },
+      ref
+   ) => {
+      return (
+         <>
+            <StyledFormHelperText valid={valid} id={inputLabel}>
```
Comment (NuraiymMamatova) [inline src/components/UI/input/Input.jsx]: why do you gave inputLabel to id attribute?

File: src/components/UI/input/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+         inputLabel,
+         placeholder,
+         text,
+         value,
+         error,
+         onChange,
+         icon,
+         name,
+         onBlur,
+         valid,
```
Comment (NuraiymMamatova) [inline src/components/UI/input/Input.jsx]: You get error props but don't use it. Please replace valid with error.

File: src/components/UI/input/Input.jsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+            <StyledFormHelperText valid={valid} id={inputLabel}>
+               {text}
+            </StyledFormHelperText>
+            <StyledOutlinedInput
+               value={value}
+               name={name}
+               bordercolor={borderError?.toString()}
+               onChange={onChange}
+               placeholder={placeholder}
+               aria-describedby={inputLabel}
```
Comment (NuraiymMamatova) [inline src/components/UI/input/Input.jsx]: What is it? Why you use it?

File: src/components/UI/input/Input.jsx (hunk 3 above)
Comment (NuraiymMamatova) [reply src/components/UI/input/Input.jsx]: Helper text should shown from bottom of component

File: src/components/UI/input/Input.jsx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+   '&:hover': {
+      webkitBoxShadow: '0 0 0 30px white inset !important',
+   },
+}))
+
+const StyledErrorIcon = styled(ErrorIcon)(({ valid }) => ({
+   color: valid ? 'red' : '',
+}))
+const StyledIconButton = styled(IconButton)`
+   margin-right: -17px;
```
Comment (NuraiymMamatova) [inline src/components/UI/input/Input.jsx]: please don't use margins

**Review by Satybaldyeva — APPROVED** [review]

**Review by NuraiymMamatova — APPROVED** [review]

**Pull request merged**